pkg/k8s/client.go

```diff
@@ -1699,3 +1746,19 @@ func (c *Client) GetHealth() Health {
 	}
 	return health
 }
+
+// GetExternalClient constructs a Client for accessing an external cluster.
+// It uses the given CA, clustername and cluster host of the cluster API endpoint
+// to configure connectivity.
+func GetExternalClient(logger logger.Logger, clusterName string, clusterHost string, cacert *certs.Certificate) *Client { //nolint interfacer
+	return &Client{
+		Logger: logger,
+		GetKubeConfigBytes: func() ([]byte, error) {
+			kubeConfig, err := CreateKubeConfig(clusterName, cacert, clusterHost, "system:masters", "admin", 24*7*time.Hour)
```

Comment:
What happens after 7 days ?

Reply:
~~This one is cool - it doesn't get written to disk and is ephemeral. Only used in this one session to connect to external cluster.~~
This now unused - since we no longer create remote RBAC,

And this does apply to the kubeconfig below.